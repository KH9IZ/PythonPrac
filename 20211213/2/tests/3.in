[152, 157, 229, 136, 61, 184, 271, 136, 124, 176, 131, 197, 221, 25, 72, 125, 106, 282, 53, 169, 69, 173, 59, 17, 263, 46, 137, 289, 281, 262, 104, 128, 78, 90, 223, 67, 191, 289, 250, 170, 178, 79, 198, 267, 281, 275, 187, 187, 27, 114, 244, 225, 166, 60, 153, 186, 297, 40, 178, 39, 177, 279, 277]