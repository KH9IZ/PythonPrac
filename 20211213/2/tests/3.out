[17, 25, 27, 39, 40, 46, 53, 59, 60, 61, 67, 69, 72, 78, 79, 90, 104, 106, 114, 124, 125, 128, 131, 136, 136, 137, 152, 153, 157, 166, 169, 170, 173, 176, 177, 178, 178, 184, 186, 187, 187, 191, 197, 198, 221, 223, 225, 229, 244, 250, 262, 263, 267, 271, 275, 277, 279, 281, 281, 282, 289, 289, 297]